shared/devices/index.render.desktop.js

```diff
@@ -1,160 +1,186 @@
-// TODO flow type this file
+/* @flow */
 import React, {Component} from 'react'
+import {Box, Text, Icon} from '../common-adapters'
+import {globalStyles, globalColors} from '../styles/style-guide'
+import type {Props as IconProps} from '../common-adapters/icon'
 
-import HardwarePhoneIphone from 'material-ui/lib/svg-icons/hardware/phone-iphone'
-import HardwareComputer from 'material-ui/lib/svg-icons/hardware/computer'
-import CommunicationVpnKey from 'material-ui/lib/svg-icons/communication/vpn-key'
-import ActionNoteAdd from 'material-ui/lib/svg-icons/action/note-add'
+import type {Props} from './index'
+import type {Device as DeviceType} from '../constants/types/flow-types'
 
-import moment from 'moment'
-import commonStyles from '../styles/common'
+class RevokedHeader extends Component {
```

Comment:
prefer this:

```js
type State = {expanded: boolean}

class RevokedHeader extends Component<void, Props, State> {
  state: State;
  ...
}
```

because if you use the Component with generics like `Component<DefaultProps, Props, State>` then you can do really cool stuff like: [PropsOf](https://github.com/keybase/client/blob/master/shared/constants/types/more.js#L22) and have better types!